frontend/components/Merchant/Header/index.js

```diff
@@ -1,19 +1,20 @@
 // import { Link } from 'react-router-dom';
-import HeaderSection from '../../../../components/HeaderSection';
-import Sign from '../../../../components/Sign';
-import { getMerchantAddress } from '../../../../utils/utils';
+import HeaderSection from '../../HeaderSection';
+import Sign from '../../Sign';
+import { getMerchantAddress } from '../../../utils/utils';
+import Link from 'next/link';
 
 const Header = ({ isSignedIn, programExists }) => {
   return (
     <HeaderSection>
       {programExists && (
         <div className="mr-4">
-          <a
-            href={`/customer/${getMerchantAddress()}`}
+          <Link
+            href={`/customer?program=${getMerchantAddress()}`}
             className="rounded-md bg-indigo-500 px-3.5 py-3 text-base leading-7 text-white shadow-sm hover:bg-indigo-600 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"
           >
             Open Customer App
-          </a>
+          </Link>
```

Comment:
What is the reason to change `{`/customer/${getMerchantAddress()}`}` to `{`/customer?program=${getMerchantAddress()}`}` ?

Reply:
hmm... I'm not sure why that changed, I'll change it back. Thanks for catching that!

Reply:
@marcinbodnar it looks like this change is correct. 

Both routes `/customer/[program]` and `/customer?program=[program]` route to `/pages/customer/[program].js`. Since I removed the `[program].js` dynamic route, I had to change everything to use the query parameter instead.

Reply:
Right, what I wonder about is why we changed it to use a query parameter instead?

Reply:
Gotcha. We need to build this app as a static site to host on IPFS. 

In order to build as a static site with a dynamic route, you must provide a list of static props to generate each possible page that the dynamic route can represent. If we wanted to keep the dynamic route `[program].js`, we would need to rebuild the app every time a customer adds a loyalty program to the contract. This didn't seam ideal, so I switched the dynamic route to use query params instead.